src/scancode/cli.py

```diff
@@ -209,6 +209,7 @@ class ScanCommand(utils.BaseCommand):
 formats = ('json', 'html', 'html-app',)
 
 def validate_formats(ctx, param, value):
+    value = value.lower()
```

Comment:
You're absolutely right. However, I "Inverted" the logic to reduce indentation and the number of code branches.